.gitignore

```diff
@@ -15,4 +15,7 @@ dist/*
 *.egg-info
 .pytest_cache/*
 test.mp4
-test.txt
\ No newline at end of file
+test.txt
+tests/__pycache__/*
+TikTokApi/__pycache__
+build/*
```

Comment:
I think if you add `*/__pycache__` here, that will cover all `__pycache__` directories